llvm/lib/Transforms/Vectorize/VPlanValue.h

```diff
@@ -89,7 +96,9 @@ class LLVM_ABI_FOR_TEST VPValue {
   enum {
     VPValueSC, /// A generic VPValue, like live-in values or defined by a recipe
                /// that defines multiple values.
-    VPVRecipeSC /// A VPValue sub-class that is a VPRecipeBase.
+    VPVRecipeSC,     /// A VPValue sub-class that is a VPRecipeBase.
+    VPRegionValueSC, /// A VPValue sub-class that is defined by a region, like
```

Comment:
```suggestion
    VPVRegionSC, /// A VPValue sub-class that is defined by a region, like
```
more consistent with `VPVRecipeSC`?